src/wallet/wallet.h

```diff
@@ -922,7 +922,7 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     CAmount GetNeedsToBeAnonymizedBalance(CAmount nMinBalance = 0) const;
     CAmount GetDenominatedBalance(bool unconfirmed=false) const;
 
-    bool GetBudgetSystemCollateralTX(CWalletTx& tx, uint256 hash, CAmount amount, bool fUseInstantSend);
+    bool GetBudgetSystemCollateralTX(CWalletTx& tx, uint256 hash, CAmount amount, bool fUseInstantSend, COutPoint outpoint=COutPoint()/*defaults null*/);
```

Comment:
Same and no need for default here imo, i.e.

```suggestion
    bool GetBudgetSystemCollateralTX(CWalletTx& tx, uint256 hash, CAmount amount, bool fUseInstantSend, const COutPoint& outpoint=COutPoint()/*defaults null*/);
```
or simply
```suggestion
    bool GetBudgetSystemCollateralTX(CWalletTx& tx, uint256 hash, CAmount amount, bool fUseInstantSend, const COutPoint& outpoint);
```

Reply:
was defaulted initially due to not being based on https://github.com/dashpay/dash/pull/2480 and the function removed in https://github.com/dashpay/dash/pull/2480 called this function and I didn't want to have to pass that parameter in that function

Reply:
Well, if you want defaults for `fUseInstantSend` then you'll _have to_ provide defaults for `outpoint` too, but that's, again, not related to this PR ;) I have no strong preferences for defaults here, that's why I specified both options :)